java/feathub-udf/src/main/java/com/alibaba/feathub/udf/PostSlidingWindowExpiredRowHandler.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 The Feathub Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.feathub.udf;
+
+import org.apache.flink.types.Row;
+import org.apache.flink.util.Collector;
+
+import java.io.Serializable;
+
+/** Interface for handling post sliding window expired row. */
+public interface PostSlidingWindowExpiredRowHandler extends Serializable {
```

Comment:
Can you explain another possible implementation of this interface? Maybe add a TODO here?